.gitignore

```diff
@@ -27,3 +27,4 @@ coverage
 build/
 dist/
 
+newrelic.js
```

Comment:
Good job .gitignoring this, you may also want to .gitignore your newrelic_agent.log file that gets generated when you connect to new relic? I was told via a PR that was also good practice